.gitignore

```diff
@@ -9,6 +9,7 @@
 /*/*.template*.json
 **/html_docs
 /*/_meta/kibana.generated
+/x-pack/*/_meta/kibana.generated
```

Comment:
I'm thinking this might be better to put into https://github.com/elastic/beats/blob/master/x-pack/.gitignore?

Reply:
changed